packages/phone-number-privacy/signer/src/signing/query-quota.ts

```diff
@@ -80,25 +90,41 @@ export async function getRemainingQueryCount(
  * If the caller is not verified, they must have a minimum balance to get the unverifiedQueryMax.
  */
 async function getQueryQuota(account: string, hashedPhoneNumber?: string) {
-  if (hashedPhoneNumber && (await isVerified(account, hashedPhoneNumber))) {
-    logger.debug('Account is verified')
-    const transactionCount = await getTransactionCountFromAccount(account)
-    return (
+  let walletAddress = await getWalletAddress(account)
+  logger.debug({ account, walletAddress }, 'begin getQueryQuota')
+  if (account === walletAddress) {
```

Comment:
It's probably safest to do a normalized comparison
```suggestion
  if (account.toLowerCase() === walletAddress.toLowerCase()) {
```